src/matter/cluster/GeneralCommissioningCluster.ts

```diff
@@ -45,18 +46,26 @@ const SetRegulatoryConfigRequestT = ObjectT({
     breadcrumbStep: Field(2, UnsignedIntT),
 });
 
+/**
+ * This cluster is used to manage global aspects of the Commissioning flow.
+ */
 export const GeneralCommissioningClusterSpec = ClusterSpec(
     0x30,
     "General Commissioning",
     {
-        breadcrumb: AttributeSpec(0, UnsignedIntT),
+        breadcrumb: AttributeSpec(0, UnsignedIntT), // INT64U ??
```

Comment:
int64u can only be represented as bigint in js, so you can use UnsignedLongT type instead.